lib/readline.js

```diff
@@ -1435,12 +1005,18 @@ exports.codePointAt = internalUtil.deprecate(codePointAt,
     'Use String.prototype.codePointAt instead.');
 
 
-/**
- * Tries to remove all VT control characters. Use to estimate displayed
- * string width. May be buggy due to not running a real state machine
- */
-function stripVTControlCharacters(str) {
-  str = str.replace(new RegExp(functionKeyCodeReAnywhere.source, 'g'), '');
-  return str.replace(new RegExp(metaKeyCodeReAnywhere.source, 'g'), '');
-}
-exports.stripVTControlCharacters = stripVTControlCharacters;
+exports.emitKeypressEvents = internalUtil.deprecate(emitKeypressEvents,
```

Comment:
I think we should at least keep the style consistent here, preferably with each argument on a separate line (for better readability).